home.js

```diff
@@ -10,4 +10,195 @@ window.onload = function () {
+    'A daily selection of privately personalized reads; no accounts or sign-ups required has been the industrys standard',
+    image1: './assets/Placeholder44b.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    headerName: 'Website Protfolio',
+    description:
+    'A daily selection of privately personalized reads; no accounts or sign-ups required has been the industrys standard',
+    image1: './assets/Placeholder44b.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    headerName: 'Profesional Art',
+    description:
+    'A daily selection of privately personalized reads; no accounts or sign-ups required has been the industrys standard',
+    image1: './assets/Placeholder44b.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    headerName: 'Data Dashboard',
+    description:
+    'A daily selection of privately personalized reads; no accounts or sign-ups required has been the industrys standard',
+    image1: './assets/Placeholder44b.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    headerName: 'Website Protfolio',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required has been the industrys standard',
+    image1: './assets/Placeholder44b.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+];
+
+function showPopup() {
+  document.querySelector('.multi-cont').style.display = 'flex';
+}
+
+/* ----------------- Building Popup (Mobile) ---------------- */
+document.addEventListener('DOMContentLoaded', (event) => {
+  const popupSection = document.querySelector('.multi-cont');
+  const popupCont = document.createElement('div');
+  popupCont.className = 'multi-cont2';
+  popupSection.append(popupCont);
+  const closeSign = document.createElement('img');
+  closeSign.classList.add('closesign');
+  closeSign.src = './assets/Icon.png';
+  closeSign.alt = 'closesign';
+  closeSign.onclick = hidePopup;
+  const popupTitle = document.createElement('h3');
+  popupTitle.className = 'multi-stories';
+  const popupUl = document.createElement('ul');
+  popupUl.className = 'ul-tech1';
+  const popupImg = document.createElement('img');
+  popupImg.alt = 'project';
+  popupImg.className = 'image1';
+  const popupDescription = document.createElement('p');
+  popupDescription.className = 'multitext';
+  const liveButton = document.createElement('button');
+  liveButton.innerText = 'See Live';
+  liveButton.className = 'btn-pop';
+  const sourceButton = document.createElement('button');
+  sourceButton.innerText = 'See Source';
+  sourceButton.className = 'btn-pop';
+  popupCont.append(
+    closeSign,
+    popupTitle,
+    popupUl,
+    popupImg,
+    popupDescription,
+    liveButton,
+    sourceButton,
```

Comment:
- [ ] Please fix the below issues for the mobile screens and make sure your design matches the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=DB34JCuHQzSTpBg8-0) design.
  - Please make sure the buttons in the pop-up are aligned in a single row.
  - Please add the icons in the buttons as in the Figma design.
  - Please use the correct color for the buttons. `#FF6B00`
  - Please apply the hover effect on the buttons.
    - ![image](https://user-images.githubusercontent.com/83803414/202737731-1064132e-a236-4267-9d41-8c478884a812.png)



### Your design
![image](https://user-images.githubusercontent.com/83803414/202737190-0e41e00d-7feb-4162-9e38-eda3ed9d3654.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/202737241-963a3215-1b0f-41c5-9c9e-689283ab5f4e.png)

Reply:
I  had to change the color due to accessibility issues.